cdisc_rules_engine/services/reporting/excel_report.py

```diff
@@ -59,6 +60,23 @@ def get_export(
         )
         wb["Conformance Details"]["B3"] = f"{round(self._elapsed_time, 2)} seconds"
         wb["Conformance Details"]["B4"] = __version__
+
+        # write dataset metadata
+        datasets_data = [
+            [
+                dataset.get("filename"),
+                dataset.get("label"),
+                dataset.get("full_path"),
+                dataset.get("modification_date"),
```

Comment:
I see there is an internal modification date and an external file system modification date. Can we display both of these?

Reply:
The template only has one field for modification date, I think displaying both will require a new template, so it should be a new issue, but I will ask @nickdedonder about this tomorrow

Reply:
Per @nickdedonder including internal modification date only.